Cirrious.FluentLayout/FluentLayout.cs

```diff
@@ -153,7 +154,7 @@ private void ThrowIfSecondItemAlreadySet()
 
         public IEnumerable<NSLayoutConstraint> ToLayoutConstraints()
         {
-            var constraint = NSLayoutConstraint.Create(
+            Constraint = NSLayoutConstraint.Create(
```

Comment:
After a while I figured out how to what you say:
1. Create an array of FluentLayout yourself rather than using the extension `AddConstraints(params FluentLayout[] fluentLayouts)`
2. Call yourself `ToLayoutConstraints()` on every FluentLayout

Is that your workaround or I´m missing something?